extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeDescriptionHelper.java

```diff
@@ -0,0 +1,255 @@
+package org.schabi.newpipe.extractor.services.youtube;
+
+import static org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.getUrlFromNavigationEndpoint;
+import static org.schabi.newpipe.extractor.utils.Utils.isNullOrEmpty;
+
+import com.grack.nanojson.JsonObject;
+
+import org.jsoup.nodes.Entities;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Stack;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public final class YoutubeDescriptionHelper {
+
+    private YoutubeDescriptionHelper() {
+    }
+
+    public static final String LINK_CLOSE = "</a>";
+    public static final String STRIKETHROUGH_OPEN = "<s>";
+    public static final String STRIKETHROUGH_CLOSE = "</s>";
+    public static final String BOLD_OPEN = "<b>";
+    public static final String BOLD_CLOSE = "</b>";
+    public static final String ITALIC_OPEN = "<i>";
+    public static final String ITALIC_CLOSE = "</i>";
```

Comment:
Is it intended that these constants are public? They are not used outside the class.

Reply:
No, they should be private, thank you for noticing